src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.searchForm {
+  margin: 20px 0;
+}
+
+.searchBar {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  display: flex;
+  width: 100%;
+
+  font-family: inherit;
+
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchBar::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #404e5f;
+}
+
+.big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  background-position: 26px 25px;
+}
+
+.small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px ;
+  background-size: 11px;
+  background-position: 13px 15px ;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar:focus {
+  border-radius: 3px;
+  font-weight: 900;
+  outline: none;
```

Comment:
Use rules that will change on focus. Outline and border-radius don't change on focus

Reply:
Not fixed